external_search.py

```diff
@@ -1063,9 +1072,17 @@ def build(self, elasticsearch):
 
         # Apply any necessary sort order.
         if self.filter:
-            order = self.filter.sort_order
-            if order is not None:
-                search = search.sort(order)
+            order_fields = self.filter.sort_order
+            if order_fields:
+                search = search.sort(*order_fields)
```

Comment:
There's now always multiple order_fields, since at a minimum we order by author, title, and work ID.